frontend/src/pages/project/create.vue

```diff
@@ -21,13 +21,17 @@
 
                 <FormRow :options="templates" :error="errors.template" v-model="input.template" id="template">Template</FormRow>
 
+                <FormRow v-if="features.billing" type="checkbox" v-model="input.billingConfirmation" id="billing-confirmation">Please acknowledge that you will be charged (15 USD) for creating this project.</FormRow>
```

Comment:
```suggestion
                <FormRow type="checkbox" v-model="input.billingConfirmation" id="billing-confirmation">
                    Confirm additional charges
                    <template v-slot:description>
                        You will be charged $15/month for this project.
                    </template>
                </FormRow>
```

Counter proposal - @sammachin please lmk your preference.

<img width="383" alt="image" src="https://user-images.githubusercontent.com/51083/162243242-9d242a91-8ef8-4f35-ae58-add954a4d7b2.png">

Reply:
I'm happy with the design, only thing is we use `US$` where displaying currency just to be clear to other countries that may use $ eg CAD